From 248f4d99cf9b3c5ef167a814630792e6b945a8cb Mon Sep 17 00:00:00 2001
From: Jayachandran B <jayachandran.b@intel.com>
Date: Thu, 9 Oct 2014 16:43:44 +0530
Subject: [PATCH 05/13] ASoC: codecs: Realtek codec driver change to limit
 variable scope

Issue: GMINL-1602

The realtek codec driver has certain variables which should
only have file scope, but dont have them declared as static.
This can cause compilation issues when drivers for other
realtek codecs are added and built together.Hence this patch
fixes the declarations anticipating the need to have multiple
realtek codec drivers to be built together.

1.'hweq_param' is defined and used in rt5640_ioctl.c
 Hence prefixed the declaration with static.

2.'hp_amp_power' is defined and used in rt5640.c
 Hence prefixed the declaration with static.

Change-Id: Iea6359a1fcfa23d4524eec97c0418558b9c663e5
Signed-off-by: Jayachandran B <jayachandran.b@intel.com>
---
 sound/soc/codecs/rt5640.c       |    4 ++--
 sound/soc/codecs/rt5640_ioctl.c |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/codecs/rt5640.c b/sound/soc/codecs/rt5640.c
index 6a53b57..d4e41e8 100644
--- a/sound/soc/codecs/rt5640.c
+++ b/sound/soc/codecs/rt5640.c
@@ -1582,7 +1582,7 @@ static int rt5640_set_dmic2_event(struct snd_soc_dapm_widget *w,
 }
 
 #if USE_ONEBIT_DEPOP
-void hp_amp_power(struct snd_soc_codec *codec, int on)
+static void hp_amp_power(struct snd_soc_codec *codec, int on)
 {
 	static int hp_amp_power_count;
 
@@ -1688,7 +1688,7 @@ static void rt5640_pmd_depop(struct snd_soc_codec *codec)
 }
 
 #else
-void hp_amp_power(struct snd_soc_codec *codec, int on)
+static void hp_amp_power(struct snd_soc_codec *codec, int on)
 {
 	static int hp_amp_power_count;
 
diff --git a/sound/soc/codecs/rt5640_ioctl.c b/sound/soc/codecs/rt5640_ioctl.c
index 00dd32a..61a5975 100644
--- a/sound/soc/codecs/rt5640_ioctl.c
+++ b/sound/soc/codecs/rt5640_ioctl.c
@@ -20,7 +20,7 @@
 #include "rt5640-dsp.h"
 #endif
 
-hweq_t hweq_param[] = {
+static hweq_t hweq_param[] = {
 	{			/* NORMAL */
 	 {0}
 	 ,
-- 
1.7.9.5

